### PR TITLE
[IEEE-mismatches] Move functional hosts to main ruleset.

### PR DESCRIPTION
**Pull request by bardiharborow**

### CODE REVIEW

**Comment by J0WI** [comment]
The main ruleset is also `default_off`...

**Comment by bardiharborow** [comment]
@J0WI I've made additional tweaks, and IEEE.xml now passes the fetch test. I'm happy to enable the ruleset, but do not have acess to a IEEE account, and hence I cannot log in to check if [trac #15653](https://trac.torproject.org/projects/tor/ticket/15653) still applies. This should be safe to merge now.

File: src/chrome/content/rules/IEEE.xml [hunk 1]
```diff
@@ -130,14 +113,6 @@
 		<test url="http://www.ieee.org/go/shop" />
 		<test url="http://www.ieee.org/go/shop/" />
```
Comment (J0WI) [inline src/chrome/content/rules/IEEE.xml]: I think we can also remove these extra rules

Comment (bardiharborow) [reply src/chrome/content/rules/IEEE.xml]: Yeah those URLs seem to behave better now.

**Pull request merged**